package.json

```diff
@@ -31,6 +31,11 @@
     ]
   },
   "dependencies": {
+    "dotenv": "^10.0.0",
```

Comment:
I'm reading up on next.js and it looks this Next.js includes this, so we can remove this https://nextjs.org/docs/basic-features/environment-variables